packages/frontend/components/cong-events-card.tsx

```diff
@@ -0,0 +1,28 @@
+import {
+  Card,
+  CardContent,
+  CardDescription,
+  CardFooter,
+  CardHeader,
+  CardTitle,
+} from "@/components/ui/card";
+import ItemCards from "./item-cards";
+import { Separator } from "./ui/separator";
+
+const CongEventsCard = ({ congEventsData }: { congEventsData: any }) => {
+  return (
+    <Card>
+      <CardHeader>
+        <CardTitle>Congregation Events: </CardTitle>
+        <CardDescription>Upcoming congregation events.</CardDescription>
+      </CardHeader>
+      {/* CO visits, assemblies, conventions etc */}
+      <CardContent>
+        <ItemCards data={congEventsData} />
+      </CardContent>
+      <CardFooter></CardFooter>
+    </Card>
+  );
+};
+
+export default CongEventsCard;
```

Comment:
Avoid default exports because of difficulties with auto-imports and naming inconsistencies.

Directly exporting allows you to force use of the correct name.